lamp/tasks/create_certificate.yml

```diff
@@ -10,7 +10,7 @@
   with_items:
       - section: alternate_names
         option: DNS.1
-        value: '{{ hostname }}'
+        value: '{{ ssl_cert_name }}'
```

Comment:
I think a better approach here would be to pull out the DNS assignments into their own task that loops through the hosts var and grabs the hostnames since we have them all. The 1-based loop index required for the `DNS.{n}` value should be available as `loop.index`.